src/core/client/count/injectJSONPCallback.ts

```diff
@@ -2,6 +2,8 @@ import { CountJSONPData } from "coral-common/types/count";
 import { COUNT_SELECTOR } from "coral-framework/constants";
 import getPreviousCountStorageKey from "coral-framework/helpers/getPreviousCountStorageKey";
 
+const TEXT_CLASS_NAME = "coral-count-text";
```

Comment:
we can't put this in `src/core/common` because then the bundler will try and pull in too much of `src/core/common` into the embed and count script bundles. This is an issue with webpack/terser that I can't fix. So I'm just ignoring DRY principles for now and having this duplicated both here and in the `.../server/app/handlers/api/story/count.ts`